docs/topic/billing/tools.md

```diff
@@ -0,0 +1,57 @@
+# Cloud billing tools
+
+There are a few tools we can use to make cloud billing easier to understand,
+reason about and use.
+
+## Using tags to separate costs
+
+Cloud billing reports by default allow us to see how much each **service**
+was charged for - for example, we could know that VMs cost us $589.
+But wouldn't it be nice to know how much of that $589 was for core nodes,
+how much was for user nodes, and how much for dask workers? Or
+hypothetically, if we had multiple profiles that were accessible only to
+certain subsets of users, which profiles would cost how much money?
+
+This is actually possible if we use **cost tags**, which can be attached
+to *most* cloud resources (**not** Kubernetes resources). Once attached
+to specific sets of resources, you can filter and group by them in the
+web reporting UI or programmatically in the billing export. This will
+count *all* costs emanating from any resource tagged with that tag. For
+example, if a node is tagged with a particular tag, the following separate
+things will all have that tag associated:
+
+1. The amount of memory allocated to that node
+2. The amount of CPU allocated to the node
+3. (If tagged correctly) The base disk allocated to that node
+4. Any network costs accrued by actions of processes on that node (subject
+   to some complexity)
```

Comment:
Hmm, I do think it's important to know that some amount of memory / CPU / disk is allocated by the cloud provider to the node because of the 'hyperconvergence' model that most cloud providers use. However, I realize that that is just not described in this document at all, and I think diving into hyperconvergence would be a little bit too detailed for this document.

I agree that as is this is a bit odd wording, I'll try to clean this up.

Reply:
Upon further thinking, I have just accepted your suggestion as is - I don't think hyperconvergence related wording is essential or important in this context. Thanks @consideRatio.